### ISSUE TITLE
-Wall added after FLAGS

### ISSUE THREAD

**Issue by andrewbest** [VCVRack/Rack#742]
I cannot suppress a warning.

**Comment by AndrewBelt** [comment]
Yeah, I need a way to remove flags in the plugin Makefile somehow.

**Comment by andrewbest** [comment]
I'd like -Wall with the ability to use -Wno.

**Comment by AndrewBelt** [comment]
Something like

```Makefile
OMIT_FLAGS += -Wall
```
which would do a set difference on `FLAGS` before passing to the compiler?

**Comment by andrewbest** [comment]
I would be happy with just having the FLAGS come later.

**Comment by AndrewBelt** [comment]
Can't you just add FLAGS += -Wno-... after including plugin.mk? Haven't tested this

**Comment by andrewbest** [comment]
Yes, it worked! Thanks!

**Comment by AndrewBelt** [comment]
Oh cool. I suppose that's because of delayed evaluation of Makefile variables.
Not the prettiest solution, but it works.
